CONTRIBUTING.rst

```diff
@@ -36,82 +42,63 @@ be solved.
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-There are three Python tools required to develop, test, and release ics.py:
+* Clone latest development release
+
+.. code-block:: bash
 
-* `poetry <https://python-poetry.org/>`_ for managing virtualenvs, dependencies,  building, and publishing the package.
-* `tox <https://tox.readthedocs.io/>`_ for running the testsuite and building the documentation.
-* `bump2version <https://pypi.org/project/bump2version/>`_ to help with making a release.
+   git clone git@github.com:ics-py/ics-py.git
 
-Their respective configuration files are :file:`pyproject.toml`, :file:`tox.ini` and :file:`.bumpversion.cfg`.
-Install the tools via pip:
+* Install `Hatch <https://hatch.pypa.io/latest/>`_
 
 .. code-block:: bash
 
-    $ pip install tox poetry bump2version --user
-
-.. note::
-    If you want to develop using multiple different Python versions, you might want to consider the
-    `poetry installer <https://python-poetry.org/docs/#installation>`_.
+   pip install hatch
 
-    Poetry will automatically manage a virtualenv that you can use for developing.
-    By default, it will be located centrally in your home directory (e.g. in :file:`/home/user/.cache/pypoetry/virtualenvs/`).
-    To make poetry use a :file:`./.venv/` directory within the ics.py folder, use the following config:
+* Open shell
 
-    .. code-block:: bash
+.. code-block:: bash
 
-        $ poetry config virtualenvs.in-project true
+   hatch shell
 
-Now you are ready to setup your development environment using the following command:
+* Run python
 
 .. code-block:: bash
 
-    $ poetry install
-
-This will create a new virtualenv and install the dependencies for using ics.py.
-Furthermore, the current source of the ics.py package will be available similar to running :command:`./setup.py develop`.
-To access the virtualenv, use :command:`poetry run python` or :command:`poetry shell`.
-The :file:`poetry.lock` file locks the versions of dependencies in the development environment set up by poetry. This ensures that such an environment is the same for everyone.
-The file :file:`poetry.lock` is only read by poetry and not included in any distributions. These restrictions don't apply when running :command:`pip install ics`.
-As tox manages its own environments and doesn't read the lock file, it installs the latest versions of dependencies for testing.
-More details on the poetry side can be found in the `poetry documentation <https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control>`_.
+   hatch run python
 
-If you made some changes and now want to lint your code, run the testsuite, or build the documentation, run tox.
-You don't have to worry about which versions in which venvs are installed and whether you're directly testing against the sources or against a built package, tox handles all that for you:
+* Lint, run the testsuite or build the documentation
 
 .. code-block:: bash
 
-    $ tox
+   hatch run tox
```

Comment:
If i follow your instructions on a fresh clone of the repo, this does not work, i guess it's missing a install step somewhere (i'm not familiar with hatch yet)
```
$ hatch run tox
/bin/sh: tox: command not found
```

Reply:
ah you are right, I had accidentally met the tox dependency already.
fixed with https://github.com/ics-py/ics-py/pull/357/commits/8f89cc660e72b3ea854314d109efaa4866cf0454